app/src/test/java/com/amaze/filemanager/asynchronous/asynctasks/compress/AbstractCompressedHelperTaskTest.kt

```diff
@@ -83,4 +76,13 @@ abstract class AbstractCompressedHelperTaskTest {
             )
         }
     }
+
+    companion object {
+        val emptyCallback = object :
```

Comment:
This should be `@JvmStatic` maybe? So that it doesn't create an extra object in runtime.